Assets/Shopify/UIToolkit/Shops/Debug/DebugSingleProductShop.cs

```diff
@@ -44,6 +45,10 @@ public class DebugSingleProductShop : MonoBehaviour, ISingleProductShop {
             LogEvent("OnCartQuantityChanged", newQuantity);
         }
 
+        public void OnCartItemsChanged(List<CartItem> cartItems) {
+            LogEvent("OnCartItemsChanged", cartItems);
+        }
```

Comment:
This event should exist in the MultiProduct shop but interface I don't think it should exist in the single product shop interface, as the purpose of the single product shop is to immediately buy a single product once.

Reply:
I guess along those same lines we should also remove `OnCartQuantityChanged`?